pkgs/development/libraries/openvino/default.nix

```diff
@@ -0,0 +1,125 @@
+{ lib
+, autoPatchelfHook
+, stdenv
+, fetchgit
+, fetchurl
+, cmake
+, git
+, protobuf
+, tbb
+, opencv
+, unzip
+, shellcheck
+, python
+, cython ? null
+, pybind11 ? null
+, enablePython ? false
+}:
+
+
+let
+  onnx_src = fetchgit {
```

Comment:
Please convert this into fetchFromGitHub

Reply:
Done!